### ISSUE TITLE
preset_testCases

### ISSUE THREAD

**Issue by Yuanzizizi** [mit-acl/cadrl_ros#22]
Hi, sorry to bother you again.

I find the test cases (**preset_testCases**, num_agents:10) you used. I run again using the checkpoint file([gym-collision-avoidance](https://github.com/mit-acl/gym-collision-avoidance)/[gym_collision_avoidance](https://github.com/mit-acl/gym-collision-avoidance/tree/release/gym_collision_avoidance)/[envs](https://github.com/mit-acl/gym-collision-avoidance/tree/release/gym_collision_avoidance/envs)/[policies](https://github.com/mit-acl/gym-collision-avoidance/tree/release/gym_collision_avoidance/envs/policies)/[GA3C_CADRL](https://github.com/mit-acl/gym-collision-avoidance/tree/release/gym_collision_avoidance/envs/policies/GA3C_CADRL)/[checkpoints](https://github.com/mit-acl/gym-collision-avoidance/tree/release/gym_collision_avoidance/envs/policies/GA3C_CADRL/checkpoints)/IROS18/network_01900000) you provided with vpref=1.0, DT=0.2. But it fails. I reduce the radius of the agent from 0.5 to 0.3, then it works but with a slightly different trajectory.
![image](https://user-images.githubusercontent.com/105049539/195141454-dfc5627e-45ad-4346-856e-3c85393a440f.png)

![image](https://user-images.githubusercontent.com/105049539/195143755-d8b26257-a992-4240-a70b-674c9278f465.png)
![1665502962257](https://user-images.githubusercontent.com/105049539/195143782-c1e963ee-8355-471f-b5e5-9dce1bd2e2a9.jpg)

Is this normal or am I using the wrong parameter file?
Thanks for your help~

**Comment by mfe7** [comment]
Does it work with dt=0.1? I recall that we used dt=0.2 in training to expedite the process but dt=0.1 in evaluation

**Comment by Yuanzizizi** [comment]
No. I tried 0.1 and 0.01, the agent moves slowly but has the same trajectory as 0.2. am I using the wrong parameter file?
(Why do you use different DT in training and evaluation? )

**Comment by mfe7** [comment]
Hmm that does seem like the right checkpoint file. I can try to run this again locally and let you know if I see the same thing.

We used a smaller dt in evaluation to match what we were running on the hardware implementation for real-time performance, but we used a larger dt in training because the computation time was prohibitive to go much smaller back several years ago when we were training these policies.

**Comment by mfe7** [comment]
I'm running the `SmallTestSuite` config with 2 agents locally. I'm able to replicate the SA-CADRL trajectories, but there must be something that's changed with the GA3C-CADRL-10 policy since generating those figures, because I am seeing different behavior even for the simple 2-agent scenarios. I can try to spend a little more time looking around, but that code has undergone a lot of changes, so I'm not sure how easy it will be to track down.

I also confirmed in a really old other repo that we were indeed using radius = 0.5 for that 10-agent circle case you tried.

**Comment by Yuanzizizi** [comment]
Thank you for your help ☺. If you succeed, could you please share your test code with me, so I can know why I failed?

**Comment by mfe7** [comment]
Hi - I spent some time digging up and running old versions of the code (from 2018) and am not having any success. My current hypothesis is that there might be a long lost checkpoint file that was used in generating the results, but accidentally overwritten sometime later on.
